packages/docusaurus-utils/src/webpackUtils.ts

```diff
@@ -0,0 +1,598 @@
+/**
```

Comment:
i think something went wrong in merge here, this file should be removed

Reply:
I still plan to work on this. This file was created by RDIL to resolve the circular dep between core and mdx-loader. For now I'm trying to make the site build first and then moving the core/webpack/utils to this file

Reply:
This will be handled by #6047